test/unit/state/siteUtilTest.js

```diff
@@ -286,6 +286,14 @@ describe('siteUtil', function () {
           assert.equal(!!processedSites.getIn([0, 'lastAccessedTime']), true)
           assert.deepEqual(processedSites.getIn([0, 'tags']).toJS(), [])
         })
+        it('returns true for bookmarked entries', function () {
+          const siteDetail = Immutable.fromJS({
+            location: testUrl1,
+            lastAccessedTime: 123,
+            tags: [siteTags.BOOKMARK]
+          })
```

Comment:
ok thanks, added one for oldSites and one for newSites. Case for `lastAccessedTime = 0` [was already covered](https://github.com/cezaraugusto/browser-laptop/blob/5f63ca3fa881e3602f68e37058792aa87a78c233/test/unit/state/siteUtilTest.js#L284) by @darkdh, since it's the same case AFAIU, I left it as-is.